README.md

```diff
@@ -164,6 +164,17 @@ class Store {
 }
 ```
 
+## MobX v6
+
+In order to become compatible with modern ES standards, decorators are not used by default in MobX v6. It still supports decorators, but they are not recommended for greenfield projects and require some extra setup as opposed to previous versions.
```

Comment:
Decorators require extra setup for React/Vue projects, but in Angular decorators are enabled by default. But I agree about the general recommendation to avoid decorators 👍